src/tool/cppwinrt/strings/base_meta.h

```diff
@@ -121,11 +121,15 @@ namespace winrt::impl
     };
 
     template <typename T>
+#ifdef __IUnknown_INTERFACE_DEFINED__
 #ifdef __clang__
     const guid guid_v{ __uuidof(T) };
 #else
     constexpr guid guid_v{ __uuidof(T) };
 #endif
+#else
```

Comment:
I think these should all be inline as well.

Reply:
Right.

Reply:
Fixed.